pkg/cmd/codespace/list.go

```diff
@@ -86,6 +86,7 @@ func (a *App) List(ctx context.Context, opts *listOptions, exporter cmdutil.Expo
 		return cmdutil.NewNoResultsError("no codespaces found")
 	}
 
+	//nolint:staticcheck // SA1019: utils.NewTablePrinter is deprecated: use internal/tableprinter
```

Comment:
Should the team create an issue to migrate these, or are you planning to tackle this as part of some other work?

cc @cmbrose (not sure who's "owning" cs CLI these days)